src/combobox/ComboBox.ts

```diff
@@ -38,15 +39,14 @@ import * as iconCss from '../common/styles/icons.m.css';
  * @property results            Results for the current search term; should be set in response to `onRequestResults`
  * @property value              Value to set on the input
  */
-export interface ComboBoxProperties extends ThemedProperties {
+export interface ComboBoxProperties extends ThemedProperties, LabeledProperties {
 	clearable?: boolean;
 	disabled?: boolean;
 	getResultLabel?(result: any): string;
 	id?: string;
-	inputProperties?: TextInputProperties;
+	inputProperties?: InputProperties;
```

Comment:
doesn't this still need to be `TextInputProperties`? It should be valid to add `type` to the input.